### DISCUSSION TITLE
Drag and Drop

### DISCUSSION THREAD

**Discussion by Elquisllei** [givanz/VvvebJs#369] [General]
Olá tudo bem, estou com uma dificuldade em copiar todos da pasta selecionada

Bom é o seguinte tenho um codigo html onde selecione um modelo de site para poder criar na base selecionada, ele criar ate então, mas ele esta pegando somente o index.html

Mas quero que ele salva junto ao novo projeto o css, js, img, images, assets e fonts, quando faço os ajuste para que pega tudo isso ele me da erro ao salvar

Vou mostra parte do html onde faz a seleção dos modelos no html

`

Nova página
Modelo
        </label>      
        <div class="col-sm-9 input">
            <div>    
                <select class="form-select" name="startTemplateUrl">        
                    <option value="themes/modelo-branco/branco-template.html">Modelo em branco</option>        
                    <option value="themes/modelo1/index.html">Modelo 1 de L2</option>       
                    <option value="themes/modelo2/index.html">Modelo 3 de L2</option>
                    <option value="themes/modelo3/index.html">Modelo 3 de L2 </option>                        
                </select>    
            </div>
        </div>     
    </div>
    <div class="mb-3 row" data-key="href">     
         <label class="col-sm-3 col-form-label">Nome da página</label>      
        <div class="col-sm-9 input">
            <div>   
                <input name="title" type="text" value="Minha página" class="form-control" placeholder="Minha página" required>  
            </div>
        </div>     
    </div>        
    <div class="mb-3 row" data-key="href">     
         <label class="col-sm-3 col-form-label">Nome do arquivo</label>      
        <div class="col-sm-9 input">
            <div>   
                <input name="file" type="text" value="my-page.html" class="form-control" placeholder="index.html" required>  
            </div>
        </div>     
    </div>       
    <div class="mb-3 row" data-key="href">     
         <label class="col-sm-3 col-form-label">Salvar na pasta</label>      
        <div class="col-sm-9 input">
            <div>   
                <input name="folder" type="text" value="my-pages" class="form-control" placeholder="/" required>  
            </div>
            
        </div>     
    </div>
  </div>
  <div class="modal-footer">
    <button class="btn btn-secondary btn-lg" type="reset" data-bs-dismiss="modal"><i class="la la-times"></i> Cancelar</button>
    <button class="btn btn-primary btn-lg" type="submit"><i class="la la-check"></i> Criar página</button>
  </div>
</div>   
A ideia aqui é salvar tudo que tiver depois do themes/demo1/

quando ele salva so salva

my-pasta/index.html

e quando for salva ele salva dentro de um pasta Projetos/MeuSite1

Projetos/MeuSite2 e assim vai

Ela teria que salvar css/ js/ img/ assets/ etc..

Ela só salva o arquivo html

**Comment by givanz** [comment]
Hi

My understanding is that you want the templates from the new page dialog to also include assets like images/css/js that will be copied when the page is created?

![templates](https://github.com/user-attachments/assets/23f56319-5bb3-4ce0-a6db-a7bca9a75d45)

For js/css libraries a cdn can be used and for for custom css and js inline `<style></style>` and `<script></script>` tags, the issue remains for other assets like images.

To work for all type of assets templates should support both .html files and folders that include everything the page needs.

**Comment by Elquisllei** [comment]
Olá, sim, sim...

Os arquivos está em .html, o que eu falo é que os outros arquivos .html que tem no modelo ele não copia e nem as pasta das imagens e os css/ js/
Ela só copia my-pages/my-pages.html

**Comment by Elquisllei** [comment]
Givan tudo bem??

Eu baixei um modelo do seu site pra testar, poco/ ele tem várias páginas em
html e as pasta css/ js/ fonts/ imagens/

E ele apenas faz a cópia de único arquivo que seria do index.html da pasta
poco/ os demais ele não cópia..

É possível fazer este ajuste??  Eu fui fazer e dá erro na hora de salvar..

Quanto custa pra poder fazer isso??

Em seg., 22 de jul. de 2024 16:31, Givan ***@***.***>
escreveu:
